### PR TITLE
Update docs build instructions

### PR DESCRIPTION
**Pull request by feorlen**
The docs build instructions in README.md should not recommend the outdated command `make all`. Instead, recommend building/viewing one docs target at a time.

Also add `venv` and `npm` to the prerequisites, they may not be installed by default on some OSes. (Specifically, Ubuntu. Tested on 22.04 LTS.)

Addresses https://github.com/minio/docs/issues/757

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -8,8 +8,10 @@ MinIO uses [Sphinx](https://www.sphinx-doc.org/en/master/index.html) to generate
 
 - Any GNU/Linux Operating System.
 - python 3.10.x and python-pip
+- python3.10-venv
 - sphinx 4.3.2
 - nodejs 14.5.0 or later
+- npm
```
Comment (feorlen) [inline README.md:14]: I don't know which versions of npm this works with, suggestions?

Comment (ravindk89) [reply README.md:14]: So locally at least 14.5.0 seems to be our base. That said I think we recently tested with Hugo...16.19.1 and it worked fine.

Since hugo seems to require 16.19 right now, and that's Node stable anyways, we could bump to that.

File: README.md [hunk 2]
```diff
@@ -8,8 +8,10 @@ MinIO uses [Sphinx](https://www.sphinx-doc.org/en/master/index.html) to generate
 
 - Any GNU/Linux Operating System.
```
Comment (djwfyi) [inline README.md:9]: Probably worth dropping this line, or at least adding MacOS to it.

File: README.md [hunk 3]
```diff
@@ -34,13 +36,13 @@ python3 -m venv venv && source venv/bin/activate
 pip install -r requirements.txt && npm install && npm run build
 ```
 
-4. Build all targets at once:
+4. Build your desired platform target.
 
 ```
-make all
+make k8s
```
Comment (djwfyi) [inline README.md:42]: k8s is going to drop a lot of sphinx warnings because of files not referenced in it.
I suggest `make linux` instead.

**Review by djwfyi — APPROVED** [review]

File: README.md (hunk 2 above)
Comment (feorlen) [reply README.md:9]: macOS 12.3 (Monterey) finally stopped shipping py 2.7, so that might be a reasonable lower bound?

Comment (djwfyi) [reply README.md:9]: Seems reasonable to me.

File: README.md [hunk 4]
```diff
@@ -6,10 +6,12 @@ MinIO uses [Sphinx](https://www.sphinx-doc.org/en/master/index.html) to generate
 
 ### Prerequisites
 
-- Any GNU/Linux Operating System.
+- Any GNU/Linux Operating System, or macOS 12.3 or later.
 - python 3.10.x and python-pip
+- python3.10-venv
 - sphinx 4.3.2
 - nodejs 14.5.0 or later
+- npm 16.19.1 or later
```
Comment (ravindk89) [inline README.md:14]: This is also what our current Hugo build for kes-docs require so we might as well bump it.

**Review by djwfyi — APPROVED** [review]
LGTM

File: README.md [hunk 5]
```diff
@@ -34,13 +36,13 @@ python3 -m venv venv && source venv/bin/activate
 pip install -r requirements.txt && npm install && npm run build
 ```
 
-4. Build all targets at once:
+4. Build your desired platform target.
 
 ```
-make all
+make linux
 ```
```
Comment (ravindk89) [inline README.md:44]: Could add multi-builds. The output gets a bit hard to parse vs serially building, but

```
make linux k8s container
```

Is my personal standard batch when adding new tutorials.

**Review by harshavardhana — APPROVED** [review]

**Review by ravindk89 — APPROVED** [review]
LGTM mod nits

File: README.md (hunk 5 above)
Comment (feorlen) [reply README.md:44]: This started working for me, unclear why but seems solid now. Adding multiple target example.
